PublicGitArchive/doc/dataset_analysis.md

```diff
@@ -0,0 +1,70 @@
+# Description of the current dataset
+
+## 1. GHTorrent: selecting the list of URLs
+
+From GHTorrent’s MySQL dump, dated from 01/01/2018.
+By querying repos with at least 50 stargazers
+
+Output: list of 190,051 URLs with duplicates  
+List of unique URLs: 187,352
+
+## 2. Collection with Borges
+
+Output of Borges:  
+
+183,015 **repositories**  
+4,337 are **indefinitely pending** for several reasons :  
+* became private  
+* not existing anymore  
```

Comment:
for private and not existing, we have the exact number, take it from the paper

Reply:
They are not in the paper, there is only " 82 removed for legal reasons" the rest "became inaccessible by the git clone time"

Reply:
now fixed